### PR TITLE
Create Engauge Digitizer manifest

### PR DESCRIPTION
**Pull request by Alexander-Wilms**

### CODE REVIEW

File: io.github.markummitchell.Engauge_Digitizer.json [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+				{
+					"type": "archive",
+					"url": "https://github.com/markummitchell/engauge-digitizer/archive/v10.4.tar.gz",
+					"sha256": "e210daec4a1ec917f1cf25f77d6dd22f7479f4c0c5287e57cbd766145df4f336"
+				}
+			],
+			"build-commands": [
+				"qmake CONFIG=release",
+				"make",
+				"cp bin/engauge /app/bin/",
```
Comment (Alexander-Wilms) [inline io.github.markummitchell.Engauge_Digitizer.json]: There's no make install yet: https://github.com/markummitchell/engauge-digitizer/issues/115

**Comment by nedrichards** [comment]
bot, build io.github.markummitchell.Engauge_Digitizer

**Comment by nedrichards** [comment]
It'd be great if you could send a PR to the appstream upstream to add OARS and release info, otherwise lgtm from a cursory look

**Comment by Alexander-Wilms** [comment]
aarch64 failed: https://flathub.org/builds/#/builders/2/builds/529

File: io.github.markummitchell.Engauge_Digitizer.json [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+		"/lib/cmake",
+		"/share/aclocal",
+		"/share/pkgconfig",
+		"*.a",
+		"*.la",
+		"*.cmake"
+	],
+	"modules": [
+		{
+			"name": "log4cpp",
```
Comment (nedrichards) [inline io.github.markummitchell.Engauge_Digitizer.json]: In order to fix the build on aarch64 you need to rm-configure: true and then add an autoreconf autogen.sh as per this example: https://github.com/flathub/com.tux4kids.tuxmath/blob/cce3487f0c53322983a9cefd531de783241d96f8/com.tux4kids.tuxmath.json#L32

Comment (Alexander-Wilms) [reply io.github.markummitchell.Engauge_Digitizer.json]: Thanks, done

Comment (TingPing) [reply io.github.markummitchell.Engauge_Digitizer.json]: Could you link the PR here for future reference.

File: io.github.markummitchell.Engauge_Digitizer.json [hunk 3]
```diff
@@ -0,0 +1,140 @@
+{
+	"app-id": "io.github.markummitchell.Engauge_Digitizer",
+	"runtime": "org.kde.Platform",
+	"runtime-version": "5.10",
+	"sdk": "org.kde.Sdk",
+	"command": "engauge",
+	"rename-icon": "engauge-digitizer",
```
Comment (TingPing) [inline io.github.markummitchell.Engauge_Digitizer.json]: You likely want `copy-icon: true` also.

File: io.github.markummitchell.Engauge_Digitizer.json (hunk 2 above)
Comment (Alexander-Wilms) [reply io.github.markummitchell.Engauge_Digitizer.json]: https://github.com/markummitchell/engauge-digitizer/pull/255

**Comment by TingPing** [comment]
@markummitchell Do you have any interest in being involved with this package?

It should be more reliably portable than the `.run` and `.appimage` packages you already offer for Linux.

**Comment by TingPing** [comment]
@Alexander-Wilms I noticed this seems to have a dependency on fuse, does that work as expected in the sandbox?

**Comment by Alexander-Wilms** [comment]
@TingPing Do you mean even when not using the AppImage? cf. https://github.com/markummitchell/engauge-digitizer/blob/master/INSTALL

**Comment by TingPing** [comment]
> Do you mean even when not using the AppImage

My mistake, didn't realize it was an appimage specific dep.

**Comment by markummitchell** [comment]
Sure, I would be happy to support flathub. I will merge today's pull request right away, and support further changes.

**Comment by nedrichards** [comment]
bot, build io.github.markummitchell.Engauge_Digitizer

**Comment by markummitchell** [comment]
Do I need to make a new release of Engauge? 

I also wanted to mention that I used to use the log4cpp library in my releases, but the log file it generates (1) was never forwarded to me as part of an error report and (2) upset the OSX sandbox system. So I created the log4cpp_null drop-in replacement for log4cpp, which is available at   https://github.com/markummitchell/log4cpp_null.

**Comment by Alexander-Wilms** [comment]
> Do I need to make a new release of Engauge?

That would be good, yes. I've updated the manifest to use log4cpp_null.

File: io.github.markummitchell.Engauge_Digitizer.json (hunk 3 above)
Comment (Alexander-Wilms) [reply io.github.markummitchell.Engauge_Digitizer.json]: Done

**Comment by markummitchell** [comment]
I have released version 10.6 of Engauge.

**Comment by TingPing** [comment]
bot, build io.github.markummitchell.Engauge_Digitizer

**Comment by TingPing** [comment]
https://flathub.org/builds/#/builders/2/builds/655

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/io.github.markummitchell.Engauge_Digitizer

**Comment by Alexander-Wilms** [comment]
Thanks!

**Comment by markummitchell** [comment]
Thank you everyone!

**Comment by TingPing** [comment]
@markummitchell BTW you can point your users here to easily install it: https://flathub.org/apps/details/io.github.markummitchell.Engauge_Digitizer

**Comment by markummitchell** [comment]
Thanks. I already have that link at https://github.com/markummitchell/engauge-digitizer/releases

**Pull request closed without merging**